src/components/common/Application.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useState, type FormEvent } from "react";
+import axios from "axios";
+import { Alert, Autocomplete, Input, TextField } from "@mui/material";
+
+const skills = [
+  'Full-stack', 'Front-end', 'Back-end', 'Dev ops', 'UI/UX design', 'Prompt engineering', 'Rust', 'Python', 'C++', 'Golang', 'Graphic Design', 'Video Editing', 'Product Management', 'Accounting', 'Law', 'Sales', 'Business Operations', 'Others'
+];
+
+const books = [
+  'Mastering Bitcoin', 'Mastering Lightning Network', 'BPD', 'LPD', 'Learning Bitcoin through Command Line', 'Programming Bitcoin', 'The Bitcoin Standard', 'Sovereign Individual', 'The Broken Money', 'The Blocksize War', 'Others'
+];
+
+const Application = (
+  { cohortName }: { cohortName: string }
```

Comment:
can we add a boolean `isRegOpen` so that we open applications only for some cohorts?